.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Remove Git metadata from Chapel release tarball
```

Comment:
According to this [SO post](http://stackoverflow.com/questions/28026767/where-should-i-place-my-global-gitattributes-file), it seems that this file should live in `.git/info/attributes`